README.rst

```diff
@@ -0,0 +1,117 @@
+MySQL database server using a simple API.
+
+Libdrizzle-redux in the sociomantic repository is a fork of the official
+libdrizzle redux at https://launchpad.net/libdrizzle.
+
+Releases
+========
+
+`Latest stable release notes
+<https://github.com/sociomantic/libdrizzle-redux/releases/latest>`_ | `Current,
+in development, release notes
+<https://github.com/sociomantic/libdrizzle-redux/blob/master/RELEASE_NOTES.md>`_ | `All
+releases <https://github.com/sociomantic/libdrizzle-redux/releases>`_
+
+Releases are handled using `GitHub releases
+<https://github.com/sociomantic/libdrizzle-redux/releases>`_. The release notes
+provided there are usually structured in 3 sections, a **Migration Instructions**,
+which are the mandatory steps the users have to do to update to a new version,
+**Deprecated** which contains deprecated functions which is recommended not to
+use but will not break any old code, and the **New Features** which are
+optional new features available in the new version that users might find
+interesting.  Using them is optional, but encouraged.
+
+These instructions should help developers to migrate from one version to
+another. The changes listed here are the steps you need to take to move from
+the previous version to the one being listed. For example, all the steps
+described in version **v1.5** are the steps required to move from **v1.4** to
+**v1.5**.
+
+If you need to jump several versions at once, you should read all the steps from
+all the intermediate versions. For example, to jump from **v1.2** to **v1.5**,
+you need to first follow the steps in version **v1.3**, then the steps in
+version **v1.4** and finally the steps in version **v1.5**.
+
+There are also sometimes *patch-level* releases, in that case there are no
+breaking changes or new features, just bug fixes, and thus, only bug fixes are
+listed in the release notes.
+
+
+v5.1.4 (2013-04-25)
+^^^^^^^^^^^^^^^^^^^
+
+Changelog
+=========
+
+There are several differences from the main libdrizzle. The following notes are
+from the libdrizzle manual
+
+Differences from Libdrizzle
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- The server-side functionality has been removed, it no longer acts as both a
+  client and server API.
+- The Drizzle prototype library functions have been removed. It now only talks
+  to MySQL compatible servers.
+- API functions have been simplified. In Libdrizzle there was a big confusion
+  over whether the application or library should be doing the allocation and
+  freeing of objects. It is now less ambiguous.
+- New binlog API added. The library can now connect as a slave or mysqlbinlog
+  client and retrieve binlog events.
+
+Differences from libmysqlclient
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+-  API is slightly different
+-  Currently missing server-side prepared statement support and protocol compression
+
+
+Dependencies
+^^^^^^^^^^^^
+
+:zlib: v1.x
+:openssl or ssl: v1.x
+
+Migration Instructions
```

Comment:
I don't think you need the migration instructions here. They can live in the release notes.